package.json

```diff
@@ -96,11 +96,11 @@
     "gulp-wait": "0.0.2",
     "gulp-webpack": "^1.5.0",
     "historyjs": "^1.8.0-b2",
-    "jest": "^29.1.1",
+    "jest": "^29.2.1",
     "jest-puppeteer": "^6.1.1",
     "node-ssi": "^0.3.2",
     "np": "^7.6.2",
-    "puppeteer": "^18.0.5",
+    "puppeteer": "^19.0.0",
```

Comment:
puppeteer had major changes between v18 and v19
https://github.com/puppeteer/puppeteer/compare/v18.0.5...v19.0.0